COMMERCE-7203 Apply mvcc to products

ctang3 opened the pull request:
https://issues.liferay.com/browse/COMMERCE-7203

ctang3 committed:
COMMERCE-7203 Make commerce-product-service MVCC enabled

ctang3 committed:
COMMERCE-7203 Build Service

ctang3 committed:
COMMERCE-7203 Fix squashed updates

ctang3 committed:
COMMERCE-7203 Versioning

naraandrade committed:
LPS-139841 Use fetch instead of axios to make requests

naraandrade committed:
LPS-139841 Update other hooks

naraandrade committed:
LPS-139841 Keep the pattern

naraandrade committed:
LPS-139841 Adjust files to follow the modifications

naraandrade committed:
LPS-139841 Remove unnused props

naraandrade committed:
LPS-139841 Remove unnecessary dependency

georgel-pop-lr committed:
LPS-141139 Remove content text from CSS styles as this is not recommended because this content is not accessible for screen readers tools. Check LPS-133314 for more reference.

brianikim committed:
COMMERCE-7047 Wrap it in aui form so it can be published properly.

Commit:
LPS-141923 Run buildService need to be more specific

naraandrade committed:
LPS-139841 Update tests

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

Preston-Crary commented:
ci:forward

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**

The pull request will automatically be forwarded to the user `brianchandotcom` if the following test suites pass:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest(master)/1069/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-30.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1069/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-28.liferay.com/job/test-portal-source-format/751/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-28.liferay.com//userContent/jobs/test-portal-source-format/builds/751/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 3 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/337ec3a08889ac565e56de221033ea4aaa9298dc">337ec3a08889ac565e56de221033ea4aaa9298dc</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/ctang3/liferay-portal/tree/COMMERCE-7203">COMMERCE-7203</a><br/>Branch GIT ID: <a href="https://github.com/ctang3/liferay-portal/commit/59b6bfc4f2058a0e8337f426b9d02f3c77bb438c">59b6bfc4f2058a0e8337f426b9d02f3c77bb438c</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-28.liferay.com/job/test-portal-source-format/751/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-28.liferay.com/userContent/jobs/test-portal-source-format/builds/751/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-28.liferay.com/job/test-portal-source-format/751/">test-portal-source-format#751</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-28.liferay.com/userContent/jobs/test-portal-source-format/builds/751/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/Preston-Crary/liferay-portal/pull/1526">Preston-Crary#1526</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1575576029">[master] ci:test:sf - ctang3 &gt; Preston-Crary - PR#1526 - 2021-11-03[11:16:07]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-28.liferay.com/job/publish-testray-report/2531/">publish-testray-report#2531</a><br/></div></div>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 11 out of 11 jobs passed</h3><h3>:x: ci:test:relevant - 22 out of 26 jobs passed in 1 hour 50 minutes</h3><details><summary>Click here for more details.</summary><br/><p>This pull is eligible for <a href="https://grow.liferay.com/share/CI+liferay-continuous-integration+GitHub+Commands#General-Commands">reevaluation</a>. When this <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)/2532">upstream build</a> has completed, using the following CI command will compare this pull request result against a more recent upstream result:<pre>ci:reevaluate:1307401_1069</pre></p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/337ec3a08889ac565e56de221033ea4aaa9298dc">337ec3a08889ac565e56de221033ea4aaa9298dc</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/f25d222450fc9e2e5d45d1018adb4978f62b44fd">f25d222450fc9e2e5d45d1018adb4978f62b44fd</a><br/>Jenkins Build URL: <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)//2531">Acceptance Upstream DXP (master) #2531</a></p><details><summary><strong>ci:test:stable - 11 out of 11 jobs PASSED</strong></summary><details><summary><strong>11 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14231/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11697/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/12683/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14232/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11978/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/12681/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11983/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11695/">test-portal-acceptance-pullrequest-batch(master)/portal-license-smoke-jdk8/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11975/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11973/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11980/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</a></li></ul></details></details><details><summary><strong>ci:test:relevant - 22 out of 26 jobs PASSED</strong></summary><div><h4>4 Failed Jobs:</h4><ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest(master)/1069/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13977/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8/0</strong></strike></a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11698/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</strong></strike></a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11694/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8/0</strong></strike></a></li></ul></div><details><summary><strong>22 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11977/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14230/">test-portal-acceptance-pullrequest-batch(master)/empty-osgi-core-dir-jdk8/0</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/12684/">test-portal-acceptance-pullrequest-batch(master)/empty-temp-dir-jdk8/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11696/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14231/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11979/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11697/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/12683/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11981/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14232/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11978/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/12682/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11976/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8/0</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/12681/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11974/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11982/">test-portal-acceptance-pullrequest-batch(master)/patching-tool-jdk8/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11983/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11695/">test-portal-acceptance-pullrequest-batch(master)/portal-license-smoke-jdk8/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11975/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11973/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11972/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11980/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</a></li></ul></details></details><h5>For more details click <a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1069/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11694/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8/0</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0/11694//consoleText">AXIS_VARIABLE=0 #11694</a><pre><code>clean-up-java-processes:
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/clean_up_java_processes.sh
[stopwatch] [run.batch.test.setup: 24:28.280 sec]
     [echo] run.batch.test.action.start.timestamp: 11-03-2021 12:37:04:698 PDT
     [echo] Checking for baseline log files.
     [echo] 
     [echo] ##
     [echo] ## /opt/dev/projects/github/liferay-portal/baseline-reports/util-taglib-8.log
     [echo] ##
     [echo] 
[beanshell] [Baseline Warning] Bundle Version Change Recommended: 8.1.4
[beanshell] 
[stopwatch] [run.batch.test.action: 9.171 sec]
     [echo] Semantic versioning is incorrect.
      [get] Getting: http://test-1-26/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0/11694//consoleText
      [get] To: /opt/dev/projects/github/liferay-portal/20211103123713870.txt
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/20211103123713870.txt
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null2015650436.properties</code></pre></div></li></ol></div></li><li><div><h5><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11698/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</a></h5><div><h6>Job Results:</h6><p>2729 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=5/11698//consoleText">AXIS_VARIABLE=5 #11698</a><ol><li><div><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=5/11698//testReport/com.liferay.blogs.asset.test/BlogsEntryAssetSearchTest/testOrderByCreateDateAsc">com.liferay.blogs.asset.test.BlogsEntryAssetSearchTest.testOrderByCreateDateAsc</a><pre><code>org.junit.ComparisonFailure: expected:&lt;{[open,liferay,social,osgi],content,life}&gt; but was:&lt;{[social,open,osgi,liferay],content,life}&gt;
	at org.junit.Assert.assertEquals(Assert.java:117)
	at org.junit.Assert.assertEquals(Assert.java:146)
	at com.liferay.asset.test.util.BaseAssetSearchTestCase.testOrderByCreateDate(BaseAssetSearchTestCase.java:1368)
	at com.liferay.asset.test.util.BaseAssetSearchTestCase.testOrderByCreateDateAsc(BaseAssetSearchTestCase.java:841)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable$3.evaluate(TestExecutorRunnable.java:353)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
	at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
	at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
	at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
	at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
	at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(Abst...</code></pre></div></li></ol></div></li></ol></div></li></ol><hr/><details><summary><strong>Failures in common with <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">acceptance upstream results</a> at f25d222450fc9e2e5d45d1018adb4978f62b44fd:</strong></summary><ol><li><div><h5><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/13977/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8/0</a></h5><div><h6>Job Results:</h6><p>3 Tests Passed.<br/>2 Tests Failed.</p></div><ol><li><div><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0/13977//consoleText">AXIS_VARIABLE=0 #13977</a><ol><li><div><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0/13977//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanOSGiLog">com.liferay.portal.log.assertor.PortalLogAssertorTest.testScanOSGiLog</a></div></li><li><div><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0/13977//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">com.liferay.portal.log.assertor.PortalLogAssertorTest.testScanXMLLog</a></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/11698/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</a></h5><div><h6>Job Results:</h6><p>2729 Tests Passed.<br/>2 Tests Failed.</p></div><ol><li><div><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1/11698//consoleText">AXIS_VARIABLE=1 #11698</a><ol><li><div><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1/11698//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">com.liferay.portal.log.assertor.PortalLogAssertorTest.testScanXMLLog</a><pre><code>junit.framework.AssertionFailedError: 
Unable to find default image scaler
	at com.liferay.portal.log.assertor.PortalLogAssertorTest.scanXMLLogFile(PortalLogAssertorTest.java:171)
	at com.liferay.portal.log.assertor.PortalLogAssertorTest$1.visitFile(PortalLogAssertorTest.java:98)
	at com.liferay.portal.log.assertor.PortalLogAssertorTest$1.visitFile(PortalLogAssertorTest.java:88)
	at java.nio.file.Files.walkFileTree(Files.java:2670)
	at java.nio.file.Files.walkFileTree(Files.java:2742)
	at com.liferay.portal.log.assertor.PortalLogAssertorTest.testScanXMLLog(PortalLogAssertorTest.java:86)
</code></pre></div></li></ol></div></li><li><div><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=3/11698//consoleText">AXIS_VARIABLE=3 #11698</a><ol><li><div><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=3/11698//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">com.liferay.portal.log.assertor.PortalLogAssertorTest.testScanXMLLog</a><pre><code>junit.framework.AssertionFailedError: 
Portlet com_liferay_commerce_account_web_internal_portlet_CommerceAccountPortlet from com.liferay.commerce.account.web_4.0.12 [1030] failed to initialize
com.liferay.portal.kernel.exception.PortalException: com.liferay.portal.kernel.exception.NoSuchResourceActionException: com.liferay.commerce.account#ADD_ACCOUNT
	at com.liferay.portal.osgi.web.portlet.tracker.internal.PortletTracker.deployPortlet(PortletTracker.java:1351)
	at com.liferay.portal.osgi.web.portlet.tracker.internal.PortletTracker.addingPortlet(PortletTracker.java:414)
	at com.liferay.portal.osgi.web.portlet.tracker.internal.PortletTracker.addingService(PortletTracker.java:179)
	at com.liferay.portal.osgi.web.portlet.tracker.internal.PortletTracker.addingService(PortletTracker.java:121)
	at org.osgi.util.tracker.ServiceTracker$Tracked.customizerAdding(ServiceTracker.java:943)
	at org.osgi.util.tracker.ServiceTracker$Tracked.customizerAdding(ServiceTracker.java:1)
	at org.osgi.util.tracker.AbstractTracked.trackAdding(AbstractTracked.java:256)
	at org.osgi.util.tracker.AbstractTracked.trackInitial(AbstractTracked.java:183)
	at org.osgi.util.tracker.ServiceTracker.open(ServiceTracker.java:321)
	at org.osgi.util.tracker.ServiceTracker.open(ServiceTracker.java:264)
	at com.liferay.portal.osgi.web.portlet.tracker.internal.PortletTracker.lambda$activate$2(PortletTracker.java:304)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:74...</code></pre></div></li></ol></div></li></ol></div></li></ol></details></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest(master)/1069/">test-portal-acceptance-pullrequest(master)#1069</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1069/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/Preston-Crary/liferay-portal/pull/1526">Preston-Crary#1526</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1575623073">[master] ci:test:relevant - ctang3 &gt; Preston-Crary - PR#1526 - 2021-11-03[11:16:04]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-30.liferay.com/job/publish-testray-report/3221/">publish-testray-report#3221</a><br/></div></div>

susanchen3 committed:
LPS-141820 Add violations of different severities

susanchen3 committed:
LPS-141820 Create paths relating to checking severity of a violation

susanchen3 committed:
LPS-141820 Add test to only filter by critical violations

susanchen3 committed:
LPS-141820 Fix SF for link

Luizcarlosqueiroz committed:
LPS-138206 Test Autom. Failing Elements Details

Luizcarlosqueiroz committed:
LPS-138206 SF

Kvale1733 committed:
LRQA-70350 Extend the @description with meaningful message

Commit:
LPS-139170 Fixes case formating

jorgediaz-lr committed:
LPS-142128 Change catch from PortalException to Exception to avoid in case of database inconsistency throwing the java.lang.IllegalArgumentException exception to the user and blocking them to login in the Liferay installation

ozysouza committed:
LPS-140896 Delete deprecated locators

ozysouza committed:
LPS-140896 Modify macros to use general locator

ozysouza committed:
LPS-140896 Delete deprecated macro

ozysouza committed:
LPS-140896 Modify IFrameDemo testcase according deleted path

naraandrade committed:
LPS-139841 Skip hard to fix tests (They will be fixed as technical debts. A lot of code needed to be commented because skip was not enough to avoid errors on CI)

ozysouza committed:
LPS-140896 Modify mock mock macro to give option to delete all emails

ozysouza committed:
LPS-140896 Modify setUp and tearDown

ozysouza committed:
LPS-140896 Refactor cases in WidgetPagesUseCase file

susanchen3 committed:
LPS-141820 Fix test for filtering by critical

susanchen3 committed:
LPS-141820 Add macro for setting filter

Commit:
LPS-140602 Use the right classloader and path to load product images

Commit:
LPS-140602 Support external urls in site navigation menus

JoyceWang08211 committed:
LRQA-70369 Add pre-config to update MFA email name and address

Commit:
LPS-141923 edit breaking change entry

peerkar committed:
LPS-112856 search-experiences-rest-impl: Add REST APIs for field mapping info

peerkar committed:
LPS-112856: gw buildRest

peerkar committed:
LPS-112856 search-experiences-rest-impl: Implement methods for field mapping info

shuyangzhou committed:
LPS-142212 Remove backend support for ext plugins.

peerkar committed:
LPS-112856 search-experiences-web: Remove obsolete field mapping info classes

Commit:
LPS-140602 Skip non JSON resources

shuyangzhou committed:
LPS-142212 Semantic versioning

Kvale1733 committed:
LRQA-70420 Extend the @description with meaningful message

Kvale1733 committed:
LRQA-70423 Flag basic UI test with @uitest

Kvale1733 committed:
LRQA-70416 Update case with proper case name

Kvale1733 committed:
LRQA-70422 Make test case steps easier to read and maintain

peerkar committed:
LPS-112856 search-experiences-rest-test: Generated tests for field mapping info

hhuijser committed:
LPS-105380 Allow checks with 'Upgrade' category that will only run when explicitly set

Kvale1733 committed:
LRQA-70340 Extend the @description with meaningful message

ramonmsp committed:
LPS-141434 Replace upper-toolbar to toolbar in package.json

ramonmsp committed:
LPS-141434 Change components structure and add translation admin

ramonmsp committed:
LPS-141434 Rename css class

ramonmsp committed:
LPS-141434 Add new props to process builder app

Commit:
LPS-141480 Fix date format inconsistency between Form Builder and Published Form

Kvale1733 committed:
LRQA-70336 Improve navigation

Kvale1733 committed:
LRQA-70337 Flag basic UI test with @uitest

Kvale1733 committed:
LRQA-70344 Update case with proper case name

arboliveira committed:
LPS-142211 search-experiences-rest-impl: rest-openapi.yaml: com.fasterxml.jackson.core.JsonParseException: Unrecognized token 'must': was expecting (JSON String, Number, Array, Object or token 'null', 'true' or 'false')

https://issues.liferay.com/browse/LPS-142211

arboliveira committed:
LPS-142211 gw buildREST

https://issues.liferay.com/browse/LPS-142211

arboliveira committed:
LPS-142211 search-experiences-service: FIX: com.fasterxml.jackson.core.JsonParseException: Unrecognized token 'must': was expecting (JSON String, Number, Array, Object or token 'null', 'true' or 'false')

https://issues.liferay.com/browse/LPS-142211

arboliveira committed:
LPS-142211 search-experiences-service: adding more OOTB SXP Elements

https://issues.liferay.com/browse/LPS-142211

arboliveira committed:
LPS-142211 search-experiences-service: Test: Make sure this never happens again

com.fasterxml.jackson.core.JsonParseException: Unrecognized token 'must': was expecting (JSON String, Number, Array, Object or token 'null', 'true' or 'false')

https://issues.liferay.com/browse/LPS-142211

Kvale1733 committed:
LRQA-70542 Skip login

arboliveira committed:
LPS-142211 search-experiences-service: acknowledge changes in DTO schema

https://issues.liferay.com/browse/LPS-142211

tinatian committed:
Revert "LPS-136498 prep next"

This reverts commit 7134722bd72af119cdfcf0ad4891ecbb822b9dfe.

tinatian committed:
Revert "LPS-136498 No longer need this as current version is using org.osgi.service.log version 1.4"

This reverts commit 585e7cb941db4b3f5ec2f698b714340766ab07de.

tinatian committed:
Revert "LPS-136498 Update reference"

This reverts commit aacd359142d52d21b89c41a8b4e80b5282bbc9b7.

tinatian committed:
Revert "LPS-136498 Auto generated"

This reverts commit a8a92d94bd66cd31392394b99367aee53b86d34b.

tinatian committed:
Revert "LPS-136498 Update patches"

This reverts commit d62d92f97d1043b0dc64b3201c1902320196f354.

tinatian committed:
Revert "LPS-136498 Upgrade org.apache.felix.scr to avoid exporting "org.osgi.util.function""

This reverts commit cff33d22cf41afd1ea36c55f4e18e938101d068f.

tinatian committed:
Revert "LPS-136498 Upgrade org.apache.felix.converter to avoid exporting "org.osgi.util.function""

This reverts commit 3fd92dac0dd241371ce381fe7c3c66fdb351b834.

tinatian committed:
Revert "LPS-136498 Remove duplicated exporting for "org.osgi.service.cm", org.apache.felix.configadmin_1.9.8.LIFERAY-PATCHED-5 will export it"

This reverts commit 3fd38b99d74aaab87a92f2026c53797cb92def76.

BryanEngler committed:
LPS-142023 search-experiences-rest-impl: Extract logic to DTOConverter and properly set localization maps in postSXPElement()

https://issues.liferay.com/browse/LPS-142023

BryanEngler committed:
LPS-142023 search-experiences-rest-api: Remove logic that was extracted to DTOConverter

https://issues.liferay.com/browse/LPS-142023

susanchen3 committed:
LPS-141820 Change key_text to key_index

susanchen3 committed:
LPS-141820 Remove for loops

michaelhashimoto committed:
LRCI-2613 Ensure the upstream branch SHA is updated as well as the rebased branch

Kvale1733 committed:
LRQA-70526 Skip login

BryanEngler committed:
LPS-142022 search-experiences-rest-impl: Extract logic to DTOConverter and properly set localization maps in postSXPBlueprint()

https://issues.liferay.com/browse/LPS-142022

BryanEngler committed:
LPS-142022 search-experiences-rest-impl: Add i18 fields to rest-openapi.yaml

https://issues.liferay.com/browse/LPS-142022

BryanEngler committed:
LPS-142022 LPS-142022: buildREST

https://issues.liferay.com/browse/LPS-142022

BryanEngler committed:
LPS-142022 search-experiences-rest-api: Remove logic that was extracted to DTOConverter

https://issues.liferay.com/browse/LPS-142022

BryanEngler committed:
LPS-142022 search-experiences-service: Use SXPBlueprintDTOConverter

https://issues.liferay.com/browse/LPS-142022

ctang3 commented:
ci:forward

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**

The pull request will automatically be forwarded to the user `brianchandotcom` if the following test suites pass:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

liferay-continuous-integration commented:
Skipping previously passed test suites:
`ci:test:sf`

JoyceWang08211 committed:
LPS-140381 Move json into macro

melodywu committed:
LRQA-71559 Disable the testcase for work around

brianchandotcom committed:
LPS-141139 SF

brianchandotcom committed:
LPS-142211 Merge error? This was removed by Petteri

brianchandotcom committed:
LPS-112856 Sort

brianchandotcom committed:
LPS-112856 Minor changes

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-forms-service 5.0.14 artifact properties

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-forms-web 5.0.21 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-forms-web 5.0.21 artifact properties

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-metrics-integration 2.0.8 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-metrics-integration 2.0.8 artifact properties

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-api 6.0.3 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-api 6.0.3 artifact properties

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-demo 2.0.6 artifact properties

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-demo 2.0.6 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-demo-data-creator-api 2.0.5 artifact properties

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-demo-data-creator-api 2.0.5 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-demo-data-creator-impl 2.0.7 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-demo-data-creator-impl 2.0.7 artifact properties

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-rest-api 11.1.2 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-rest-api 11.1.2 artifact properties

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-rest-client 3.0.9 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-rest-client 3.0.9 artifact properties

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-rest-impl 3.0.23 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-rest-impl 3.0.23 artifact properties

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-rest-spi 3.0.7 artifact properties

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-rest-spi 3.0.7 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-service 3.0.14 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-service 3.0.14 artifact properties

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-web 3.0.23 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-web 3.0.23 artifact properties

brianchandotcom committed:
artifact:ignore saml-addon-keep-alive-web 5.0.12 artifact properties

brianchandotcom committed:
artifact:ignore saml-addon-keep-alive-web 5.0.12 prep next

brianchandotcom committed:
artifact:ignore saml-api 6.0.11 prep next

brianchandotcom committed:
artifact:ignore saml-api 6.0.11 artifact properties

brianchandotcom committed:
artifact:ignore saml-impl 5.0.14 prep next

brianchandotcom committed:
artifact:ignore saml-impl 5.0.14 artifact properties

brianchandotcom committed:
artifact:ignore saml-opensaml-integration 5.3.11 artifact properties

brianchandotcom committed:
artifact:ignore saml-opensaml-integration 5.3.11 prep next

brianchandotcom committed:
artifact:ignore saml-persistence-api 8.0.3 artifact properties

brianchandotcom committed:
artifact:ignore saml-persistence-api 8.0.3 prep next

brianchandotcom committed:
artifact:ignore saml-persistence-service 5.0.20 prep next

brianchandotcom committed:
artifact:ignore saml-persistence-service 5.0.20 artifact properties

brianchandotcom committed:
artifact:ignore saml-web 5.0.23 prep next

brianchandotcom committed:
artifact:ignore saml-web 5.0.23 artifact properties

brianchandotcom committed:
artifact:ignore segments-asah-connector 3.0.12 prep next

brianchandotcom committed:
artifact:ignore segments-asah-connector 3.0.12 artifact properties

brianchandotcom committed:
artifact:ignore segments-asah-rest-api 4.2.2 artifact properties

brianchandotcom committed:
artifact:ignore segments-asah-rest-api 4.2.2 prep next

brianchandotcom committed:
artifact:ignore segments-asah-rest-client 3.0.8 artifact properties

brianchandotcom committed:
artifact:ignore segments-asah-rest-client 3.0.8 prep next

brianchandotcom committed:
artifact:ignore segments-asah-rest-impl 3.0.15 artifact properties

brianchandotcom committed:
artifact:ignore segments-asah-rest-impl 3.0.15 prep next

brianchandotcom committed:
artifact:ignore segments-dxp-web 1.0.9 prep next

brianchandotcom committed:
artifact:ignore segments-dxp-web 1.0.9 artifact properties

brianchandotcom committed:
artifact:ignore segments-experiment-web 3.0.16 artifact properties

brianchandotcom committed:
artifact:ignore segments-experiment-web 3.0.16 prep next

brianchandotcom committed:
artifact:ignore sharepoint-rest-oauth2-api 5.2.3 prep next

brianchandotcom committed:
artifact:ignore sharepoint-rest-oauth2-api 5.2.3 artifact properties

brianchandotcom committed:
artifact:ignore sharepoint-rest-oauth2-service 5.0.13 prep next

brianchandotcom committed:
artifact:ignore sharepoint-rest-oauth2-service 5.0.13 artifact properties

brianchandotcom committed:
artifact:ignore sharepoint-rest-oauth2-web 3.0.9 artifact properties

brianchandotcom committed:
artifact:ignore sharepoint-rest-oauth2-web 3.0.9 prep next

brianchandotcom committed:
artifact:ignore sharepoint-rest-repository 5.0.14 prep next

brianchandotcom committed:
artifact:ignore sharepoint-rest-repository 5.0.14 artifact properties

brianchandotcom committed:
artifact:ignore sharepoint-soap-repository 2.0.11 prep next

brianchandotcom committed:
artifact:ignore sharepoint-soap-repository 2.0.11 artifact properties

brianchandotcom committed:
artifact:ignore vldap-server 5.0.9 prep next

brianchandotcom committed:
artifact:ignore vldap-server 5.0.9 artifact properties

brianchandotcom committed:
artifact:ignore headless-commerce-machine-learning-api 5.4.1 artifact properties

brianchandotcom committed:
artifact:ignore headless-commerce-machine-learning-api 5.4.1 prep next

brianchandotcom committed:
artifact:ignore headless-commerce-machine-learning-client 1.0.2 prep next

brianchandotcom committed:
artifact:ignore headless-commerce-machine-learning-client 1.0.2 artifact properties

brianchandotcom committed:
artifact:ignore headless-commerce-machine-learning-impl 5.0.12 prep next

brianchandotcom committed:
artifact:ignore headless-commerce-machine-learning-impl 5.0.12 artifact properties

brianchandotcom committed:
artifact:ignore commerce-theme-minium-full-site-initializer 4.0.9 artifact properties

brianchandotcom committed:
artifact:ignore commerce-theme-minium-full-site-initializer 4.0.9 prep next

brianchandotcom committed:
artifact:ignore headless-commerce-punchout-api 5.2.2 prep next

brianchandotcom committed:
artifact:ignore headless-commerce-punchout-api 5.2.2 artifact properties

brianchandotcom committed:
artifact:ignore headless-commerce-punchout-impl 4.0.10 prep next

brianchandotcom committed:
artifact:ignore headless-commerce-punchout-impl 4.0.10 artifact properties

JoyceWang08211 committed:
LRQA-70367 Flag basic UI test with @uitest

anabuchmann committed:
LRQA-71562 - Quarantine TaskIsNotDisplayedAfterOrderIsDeleted test

ozysouza committed:
LPS-140896 Add step do sign in as test in tearDown

ozysouza committed:
LPS-140896 Modify testcase name

ozysouza committed:
LPS-140896 Fix instance property name and change to AssertTextNotPresent

brianchandotcom committed:
LPS-142231 add release.info.version.display.name

brianchandotcom committed:
Fix version increase

brianchandotcom committed:
artifact:ignore portal-impl 10.0.0 prep next

brianchandotcom committed:
artifact:ignore portal-impl 10.0.0 releng

brianchandotcom committed:
artifact:ignore portal-kernel 19.0.0 releng

brianchandotcom committed:
artifact:ignore portal-kernel 19.0.0 prep next

arthurchan35 committed:
LPS-142146 antisamy uses non osgi class loading,

we need to fix class loading context to its own module.

brianchandotcom committed:
LPS-142231 Here too

brianchandotcom committed:
LPS-142231 prep next

brianchandotcom committed:
LPS-142231 prep next

brianchandotcom committed:
artifact:ignore portal-web 5.0.64 prep next

brianchandotcom committed:
artifact:ignore portal-web 5.0.64 releng

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
artifact:ignore blogs-web 6.0.24 change log

brianchandotcom committed:
artifact:ignore blogs-web 6.0.24 artifact properties

brianchandotcom committed:
artifact:ignore blogs-web 6.0.24 prep next

brianchandotcom committed:
artifact:ignore commerce-frontend-taglib 13.0.6 change log

brianchandotcom committed:
artifact:ignore commerce-frontend-taglib 13.0.6 prep next

brianchandotcom committed:
artifact:ignore commerce-frontend-taglib 13.0.6 artifact properties

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-form-web 4.0.44 change log

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-form-web 4.0.44 prep next

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-form-web 4.0.44 artifact properties

brianchandotcom committed:
artifact:ignore frontend-editor-alloyeditor-web 5.0.15 change log

brianchandotcom committed:
artifact:ignore frontend-editor-alloyeditor-web 5.0.15 artifact properties

brianchandotcom committed:
artifact:ignore frontend-editor-alloyeditor-web 5.0.15 prep next

brianchandotcom committed:
artifact:ignore frontend-editor-ckeditor-web 5.0.28 change log

brianchandotcom committed:
artifact:ignore frontend-editor-ckeditor-web 5.0.28 prep next

brianchandotcom committed:
artifact:ignore frontend-editor-ckeditor-web 5.0.28 artifact properties

brianchandotcom committed:
artifact:ignore frontend-theme-admin 5.0.10 artifact properties

brianchandotcom committed:
artifact:ignore frontend-theme-admin 5.0.10 digest

brianchandotcom committed:
artifact:ignore frontend-theme-admin 5.0.10 prep next

brianchandotcom committed:
LPS-140602 Simplify and make null check consistent

brianchandotcom committed:
artifact:ignore frontend-theme-classic 5.0.17 prep next

brianchandotcom committed:
artifact:ignore frontend-theme-classic 5.0.17 digest

brianchandotcom committed:
artifact:ignore frontend-theme-classic 5.0.17 artifact properties

brianchandotcom committed:
artifact:ignore frontend-theme-dialect 5.0.4 artifact properties

brianchandotcom committed:
artifact:ignore frontend-theme-dialect 5.0.4 digest

brianchandotcom committed:
artifact:ignore frontend-theme-dialect 5.0.4 prep next

brianchandotcom committed:
artifact:ignore frontend-theme-styled 6.0.13 change log

brianchandotcom committed:
artifact:ignore frontend-theme-styled 6.0.13 artifact properties

brianchandotcom committed:
artifact:ignore frontend-theme-styled 6.0.13 prep next

brianchandotcom committed:
artifact:ignore layout-content-page-editor-web 3.0.55 change log

brianchandotcom committed:
artifact:ignore layout-content-page-editor-web 3.0.55 artifact properties

brianchandotcom committed:
artifact:ignore layout-content-page-editor-web 3.0.55 prep next

brianchandotcom committed:
artifact:ignore portal-template-soy-renderer-impl 3.0.6 prep next

brianchandotcom committed:
artifact:ignore portal-template-soy-renderer-impl 3.0.6 artifact properties

brianchandotcom committed:
artifact:ignore commerce-theme-minium 5.0.15 digest

brianchandotcom committed:
artifact:ignore commerce-theme-minium 5.0.15 prep next

brianchandotcom committed:
artifact:ignore commerce-theme-minium 5.0.15 artifact properties

brianchandotcom committed:
artifact:ignore commerce-theme-speedwell 4.0.17 prep next

brianchandotcom committed:
artifact:ignore commerce-theme-speedwell 4.0.17 artifact properties

brianchandotcom committed:
artifact:ignore commerce-theme-speedwell 4.0.17 digest

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
artifact:ignore commerce-checkout-web 5.0.18 change log

brianchandotcom committed:
artifact:ignore commerce-checkout-web 5.0.18 prep next

brianchandotcom committed:
artifact:ignore commerce-checkout-web 5.0.18 artifact properties

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
LPS-140602 Simplify

Commit:
LPS-140896 Correct full name

Commit:
LPS-140896 Sign out at site page

Commit:
LPS-140896 SF

brianchandotcom committed:
LPS-140602 why do we need this check? Shouldn't it only contain JSON files in that dir?

brianchandotcom committed:
COMMERCE-7047 Hugo SF, inline

brianchandotcom committed:
LPS-141434 Inline

brianchandotcom committed:
LPS-112856 Same thing

Commit:
LPS-140950 Add missing part

Commit:
LPS-140950 Add macro to import style book from temp folder

Commit:
LPS-140950 Add case to export and import style book

brianchandotcom committed:
LPS-142146 as used

BryanEngler committed:
LPS-142106 search-experiences-rest-impl: Add patchSXPElement() and unpack JSON from request body

https://issues.liferay.com/browse/LPS-142106

BryanEngler committed:
LPS-142106 search-experiences-rest-impl: Add hidden property to SXPElement

https://issues.liferay.com/browse/LPS-142106

BryanEngler committed:
LPS-142106 buildREST

https://issues.liferay.com/browse/LPS-142106

brianchandotcom committed:
LPS-142023 make it more readable

Commit:
LPS-142246 Fix Objects Poshi tests that update entry

BryanEngler committed:
LPS-142106 search-experiences-rest-api: Make public so that we can unpack JSON from body of REST call

https://issues.liferay.com/browse/LPS-142106

rafaellajordao committed:
LRQA-71572 Investigate CI failure of FormsParagraphField#CanSupportTables and FormsSubmissions#SuccessPageCanBeCustomized

brianchandotcom committed:
LRCI-2613 prep next

brianchandotcom committed:
LPS-141820 SF

Commit:
LRQA-71567 Added line; SF fix

ctang3 commented:
ci:test:relevant

ctang3 commented:
ci:forward

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**

The pull request will automatically be forwarded to the user `brianchandotcom` if the following test suites pass:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**
